### ISSUE TITLE
Provide more context on >50 users self-service error 

### ISSUE THREAD

**Issue by ebrodymoore** [sourcegraph/sourcegraph#14675]
Elaborate on the error: "The maximum users on a Team plan is 50. Please [contact sales](https://about.sourcegraph.com/contact/request-info/?form_submission_source=pricing-enterprise)"

![Screen Shot 2020-10-13 at 10 18 58 AM](https://user-images.githubusercontent.com/25422491/95894154-8aeebd00-0d3d-11eb-8d06-e7f465591d8b.png)

**Comment by ebrodymoore** [comment]
@Joelkw fyi in case you need to add this to a backlog or something. I was just going through my old issues and I added this one before you started getting notified by the team/web label.

**Comment by Joelkw** [comment]
@ebrodymoore Thanks! Do you have any data on how many people hit this error (or how many people on self-service are near/likely to hit this error)?

**Comment by ebrodymoore** [comment]
No, this is low priority but hopefully easy fix. If you point me to the place in the code where the error message is maybe I could even do it
